editor/src/components/canvas/canvas-strategies/post-action-options/post-action-paste.ts

```diff
@@ -349,13 +419,38 @@ export const PropsReplacedPastePostActionChoice = (
           elementPasteWithMetadata: dataWithPropsReplaces,
           targetOriginalPathTrees: postActionMenuData.targetOriginalPathTrees,
           canvasViewportCenter: postActionMenuData.canvasViewportCenter,
+          originalAllElementProps: postActionMenuData.originalAllElementProps,
           reparentStrategy: null,
           insertionPosition: null,
         },
       ),
   }
 }
 
+export function getUidsFromJSXElementChild(element: JSXElementChild): string[] {
```

Comment:
Is this used?

Reply:
Seems like it isn't, I removed it in [afd89c8](https://github.com/concrete-utopia/utopia/pull/3890/commits/afd89c8c462210d9a8e41cb72db326b0e4e72261)